k.makoyid/oop_code/Activation_Functions.py

```diff
@@ -0,0 +1,18 @@
+import numpy as np
+
+
+class ActivationFunctions:
+    @staticmethod

```

Comment:
There is no need in separate class, just delete it and leave functions as part of a module